src/DssSpell.sol

```diff
@@ -67,109 +69,113 @@ contract DssSpellAction is DssAction {
-    address internal constant RWA015_A_CUSTODY                = 0x65729807485F6f7695AF863d97D62140B7d69d83;
-    // BlockTower Input Conduits
-    address internal constant RWA015_A_INPUT_CONDUIT_URN_GUSD = 0x850e4E039B8ab0682045aCe945Ebc94E9f5A63DD;
-    address internal constant RWA015_A_INPUT_CONDUIT_JAR_GUSD = 0x0AaA36F5F87C767e9cE131cB5321446b2d9EF721;
-    address internal constant RWA015_A_INPUT_CONDUIT_URN_PAX  = 0xd70d035F723fE739EaDD9D529987D0Aaa291a8FF;
-    address internal constant RWA015_A_INPUT_CONDUIT_JAR_PAX  = 0xAd235C744C66d0850C04EDde69C83f977D6207d8;
+    //  ---------- Math ----------
+    uint256 internal constant MILLION = 10 ** 6;
 
     // ---------- Spark Proxy ----------
     // Spark Proxy: https://github.com/marsfoundation/sparklend/blob/d42587ba36523dcff24a4c827dc29ab71cd0808b/script/output/5/primary-sce-latest.json#L2
     address internal constant SPARK_PROXY = 0x4e847915D8a9f2Ab0cDf2FC2FD0A30428F25665d;
 
     // ---------- Trigger Spark Proxy Spell ----------
-    address internal constant SPARK_SPELL = 0xFBdB6C5596Fc958B432Bf1c99268C72B1515DFf0;
+    address internal constant SPARK_SPELL = 0x95bcf659653d2E0b44851232d61F6F9d2e933fB1;
+
 
     function actions() public override {
-        // ---------- Management of ConsolFreight (RWA003-A) Default ----------
-        // Forum: https://forum.makerdao.com/t/consolfreight-rwa-003-cf4-drop-default/21745
+        // ---------- Stability Scope Parameter Changes ----------
+        // MIP: https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969
+        // Increase the ETH-A Stability Fee (SF) by 0.12% from 3.58% to 3.70%.
+        DssExecLib.setIlkStabilityFee("ETH-A", THREE_PT_SEVEN_ZERO_PCT_RATE, /* doDrip = */ true);
 
-        // Set DC to 0
-        // Note: it was agreed with GovAlpha that there will be no global DC reduction this time.
-        DssExecLib.setIlkDebtCeiling("RWA003-A", 0);
-        // Call tell() on RWALiquidationOracle
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).tell("RWA003-A");
+        // Increase the ETH-B Stability Fee (SF) by 0.12% from 4.08% to 4.20%.
+        DssExecLib.setIlkStabilityFee("ETH-B", FOUR_PT_TWO_ZERO_PCT_RATE, /* doDrip = */ true);
 
+        // Increase the ETH-C Stability Fee (SF) by 0.12% from 3.33% to 3.45%.
+        DssExecLib.setIlkStabilityFee("ETH-C", THREE_PT_FOUR_FIVE_PCT_RATE, /* doDrip = */ true);
 
-        // ---------- Auth ESM on the Vow ----------
-        // Forum: http://forum.makerdao.com/t/overlooked-vectors-for-post-shutdown-governance-attacks-postmortem/20696/5
+        // Activate DC-IAM for PSM-PAX-A
+        // Maximum Debt Ceiling (line): 120M
+        // Target Available Debt (gap): 50 million DAI
+        // Ceiling Increase Cooldown (ttl): 24 hours
+        DssExecLib.setIlkAutoLineParameters("PSM-PAX-A", 120 * MILLION, 50 * MILLION, 24 hours);
```

Comment:
Missing comments like the change below.
It's fine to either have the comments or not, but they should be consistent.

Reply:
I think he refers to the inline comments here:
https://github.com/makerdao/spells-goerli/blob/eafbc51c9d366e79d487b779c005aa62636116db/src/DssSpell.sol#L108

I've also seen this inconsistency, but though it's fine 🙂